proto/internal/temporal/server/api/historyservice/v1/request_response.proto

```diff
@@ -403,7 +403,8 @@ message DescribeHistoryHostRequest {
     //ip:port
     string host_address = 1;
     int32 shard_id_for_host = 2;
-    temporal.api.common.v1.WorkflowExecution execution_for_host = 3;
+    string namespace_id = 3;
```

Comment:
Why it is `namespace` above and `namespace_id` here?

Reply:
I guess for consistency.

Reply:
They represent 2 different identifiers for namespace.  `namespace` is used whenever API takes namespace name and `namespace_id` is used whenever API takes internal uuid which represents the namespace.  Internally within the system we use namespace_id as the identifier.